tests/e2e/sms-gateway.js

```diff
@@ -357,35 +256,15 @@ describe('sms-gateway api', () => {
       expect(response.messages[1].to).toBe(messageTo2);
       expect(response.messages[1].content).toBe(messageContent2);
 
-      commonElements.goToReports();
-      helper.waitUntilReady(element(by.css('#reports-list li:first-child')));
-
-      helper.clickElement(
-        element(by.css('#reports-list li:first-child .heading'))
-      );
-      helper.waitElementToPresent(
-        element(by.css('#reports-content .body .item-summary .icon'))
-      );
+      await smsGatewayPo.showReport(savedDoc);
 
-      helper.waitForAngularComplete();
+      browser.waitForAngular();
```

Comment:
This looks like it needs an `await`:
```suggestion
      await browser.waitForAngular();
```
...or is there some other magic going on that I'm missing? Otherwise, that `smsGatewayPo.feedbackState()` from the following line is going to execute, but Angular might not be ready.

Reply:
Well truthfully that call is supposed to be happening on every action per the api docs. I likely can remove it. I'll see what happens.